study.md

```diff
@@ -160,15 +164,18 @@ What is a response? What does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+A response is feedback on whether a request to a server is or is not succussful.
+HTTP 200 - standard response that every thing is fine
+HTTP 404 - the client requested something that is not on the server.
+HTTP 503 - server is unavailable
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+curl -L www.reddit.com (currently the site seems to not allow these requests)
```

Comment:
Never seen one like that, this is a normal one

`curl --request GET https://www.reddit.com`